code/go/0chain.net/smartcontract/storagesc/handler.go

```diff
@@ -120,7 +120,7 @@ func (msc *StorageSmartContract) GetTransactionByHashHandler(
 	return &transaction, err
 }
 
-func (ssc *StorageSmartContract) GetAllocationsHandler(ctx context.Context,
+func (ssc *StorageSmartContract) GetAllocationsHandlerDeprecated(ctx context.Context,
```

Comment:
Can we add a line above it starting with `Deprecated:`, in this way static linter can give warning with deprecation ?